lib/integration/queryManager.ts

```diff
@@ -0,0 +1,30 @@
+import { logMessage } from "../logger";
+import dbConnector from "./dbConnector";
+import { QueryArrayResult } from "pg";
+
+const queryManager = {
+  async executeQuery(sql: string, params: [string]): Promise<QueryArrayResult<unknown[]>> {
+    try {
+      // Get a client
+      const client = dbConnector();
```

Comment:
Pass this in instead of instantiating here. In this way the same client can be used over and over again rather than creating a new one each time this is called.